frontend/src/components/auth.tsx

```diff
@@ -2,6 +2,9 @@ import { useEffect, useState } from "react";
 import io from "socket.io-client";
 import type { Socket } from "socket.io-client";
 
+const serverUrl = "http://localhost:5000";
+const websocketUrl = "ws://localhost:5000";
```

Comment:
hardcoded? just a reminder to put env var

Reply:
j for testing purposes - once start proper will create the relevant env variables!